src/sidebar/store/modules/groups.js

```diff
@@ -11,6 +11,14 @@ import session from './session';
  */
 
 const initialState = {
+  /**
+   * When there are entries, only `groups` with `id`s included in this list
+   * of will be displayed in the sidebar.
+   *
+   * @type {Group["id"][]|null}
+   */
+  filteredGroupIds: null,
```

Comment:
This is now `null` when not set instead of an empty Array, per reviewer suggestion.